packages/blockchain-info-components/src/Icons/CoinAccountIcon.js

```diff
@@ -58,7 +58,12 @@ const CoinAccountIcon = props => {
 
   return (
     <Wrapper {...props}>
-      <Image height='32px' name={coin.toLowerCase()} width='32px' />
+      <Image
+        alt={`${coin} ${accountType === 'ACCOUNT' ? accountType : `${accountType} account`}`}
```

Comment:
we should move this `ACCOUNT` to enum as well

Reply:
This should probably be a new enum called `CoinAccountIconEnum` or something instead of us cherry picking enums from all over. This was just a linter-caused line change and I think making this change is out of the scope of this fix. I'll add a todo though.

Reply:
I also realized that this file is a .js so the entire thing needs to be converted to typescript.